### PR TITLE
blog post about conda-forge

### PR DESCRIPTION
**Pull request by mattip**

### CODE REVIEW

**Comment by mattip** [comment]
Anyone want to take a look at the [new blog post](https://deploy-preview-77--keen-mestorf-442210.netlify.app/posts/2022/11/pypy-and-conda-forge.html), changed [download page](https://deploy-preview-77--keen-mestorf-442210.netlify.app/download.html) to emphasize using conda, or [instructions to install modules](https://deploy-preview-77--keen-mestorf-442210.netlify.app/download_advanced.html#installing-more-modules) mentioning conda?

File: pages/download.rst [hunk 1]
```diff
@@ -7,7 +7,11 @@
 .. description:
 
 
-We provide pre-compiled binaries for many platforms and OSes.
+We provide pre-compiled binaries for many platforms and OSes. There are also
+pre-compiled binaries available on `conda-forge`_. We have found conda-forge
+a convenient and cooperative community for distributing not only the
+interpreter, but many binary packages like SciPy which may not be available
+on PyPI.
```
Comment (mattip) [inline pages/download.rst:14]:
```suggestion
interpreter, but many packages like SciPy that are difficult to build and 
which do not yet have binary PyPy builds available on PyPI.
```

Comment (cfbolz) [inline pages/download.rst:14]: this is a bit ambiguous. SciPy is available on PyPI for PyPy, right? just not as a binary package.

**Review by cfbolz — COMMENTED** [review]
I like this! left one comment.

**Pull request merged**